src/CarbonAware.NewCLI/CarbonAware.NewCLI.csproj

```diff
@@ -0,0 +1,37 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
```

Comment:
I believe the main use-case for this tool is to be available on a server as another way for software to access carbon-relevant data.  Given that the large majority of software is run on linux-based servers, have we tested outputting a compatible binary for that case?

Reply:
Absolutely agree with this. We need to have a deployment pipeline with VMs to test that this works on Linux servers

Reply:
There is no need to specify the output. dotnet has flags to specify the arch type. By default it detects the arch and generates the specific bin (see https://docs.microsoft.com/en-us/dotnet/core/tools/dotnet-publish)

Reply:
This is all true but we're overriding the default binary name to be `caw` per @esgraham